examples/capricorn_contract_interaction_rest_example.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think adding a new file is necessary to demonstrate capricorn usage. Making the default example configurable (json file of the node config as cli argument for e.g.) should be enough